src/components/Bio.js

```diff
@@ -27,12 +27,10 @@ class Bio extends React.Component {
         />
         <p>
           Written by the folks at the{' '}
-          <strong>Open Source Developers' Club</strong> who live in and around
-          JIIT, Noida, India.
-        </p>
-      </div>
-    )
-  }
-}
-
+          <strong>Open Source Developers Community</strong> who live in and around JIIT, Noida, India.
```

Comment:
Thanks to sublime, the whitespace issues seem to be fixed now. Please reindent these properly and we'd be good to go.